script.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function() {
+  var ticTacToe = new Game($('table'));
+  ticTacToe.initCells();
+});
+
+function Game(element) {
+  game = this;
+  game.status = 1;
+  game.moves = []; // use to find a winner
```

Comment:
instead of using a variable game here you could use "this" for defining properties